gradoop-flink/src/test/java/org/gradoop/flink/algorithms/gelly/randomjump/KRandomJumpGellyVCITest.java

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.algorithms.gelly.randomjump;
+
+import org.apache.flink.api.java.io.LocalCollectionOutputFormat;
+import org.gradoop.common.model.impl.pojo.Edge;
+import org.gradoop.common.model.impl.pojo.Vertex;
+import org.gradoop.flink.model.GradoopFlinkTestBase;
+import org.gradoop.flink.model.impl.epgm.LogicalGraph;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test class for {@link KRandomJumpGellyVCI}
+ */
+@RunWith(Parameterized.class)
+public class KRandomJumpGellyVCITest extends GradoopFlinkTestBase {
+
+  /**
+   * Name for test-case
+   */
+  private String testName;
+
+  /**
+   * Value for maximum number of iterations for the algorithm
+   */
+  private final int maxIterations;
+
+  /**
+   * Probability for jumping to a random vertex instead of walking to a random neighbor
+   */
+  private final double jumpProbability;
+
+  /**
+   * Relative amount of vertices to visit via walk or jump
+   */
+  private final double percentageVisited;
+
+  /**
+   * Number of start vertices
+   */
+  private final int k;
+
+  /**
+   * The original graph used for testing
+   */
+  private LogicalGraph graph;
+
+  /**
+   * The result graph used for testing
+   */
+  private LogicalGraph resultGraph;
+
+  /**
+   * List for result vertices
+   */
+  private List<Vertex> resultVertices;
+
+  /**
+   * List for result edges
+   */
+  private List<Edge> resultEdges;
+
+  /**
+   * Creates an instance of the base test class.
+   *
+   * @param testName          Name for test-case
+   * @param k                 Number of start vertices
+   * @param maxIterations     Value for maximum number of iterations for the algorithm
+   * @param jumpProbability   Probability for jumping to a random vertex instead of walking to
+   *                          a random neighbor
+   * @param percentageVisited Relative amount of vertices to visit via walk or jump
+   */
+  public KRandomJumpGellyVCITest(String testName, String k, String maxIterations,
+    String jumpProbability, String percentageVisited) {
+    this.testName = testName;
```

Comment:
You are using the name of the test to decide it's behaviour, which seems a bit confusing. Other opinions on that?

Reply:
I'll remove the Parameterizing at all and use TestCases for each Parameter-Constellation.